types/hash-chain/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "target": "ES2015",
```

```suggestion
        "target": "es6",
```
same, probably the reason of 'check config' tag

---

done